widgetsnbextension/src/widgetarea.js

```diff
@@ -96,10 +92,6 @@ WidgetArea.prototype._create_elements = function() {
                 for (var i = 0; i < that.widget_views.length; i++) {
                     var view = that.widget_views[i];
                     view.remove();
-
-                    // Remove widget live events.
-                    view.off('comm:live', that._widget_live);
-                    view.off('comm:dead', that._widget_dead);
```

Comment:
@SylvainCorlay I read quickly and mistaked these lines as the old unregister point.